deltachat-ffi/deltachat.h

```diff
@@ -2637,21 +2619,22 @@ void           dc_accounts_unref                (dc_accounts_t* accounts);
 uint32_t       dc_accounts_add_account          (dc_accounts_t* accounts);
 
 /**
- * Add a new closed account to the account manager.
- * Internally, dc_context_new_closed() is called using a unique database-name
- * in the directory specified at dc_accounts_new().
+ * Add a new account with a password to the account manager.
```

Comment:
I would call this "passphrase" for consistency with SQLCipher documentation and the rest of the API.